pkg/collector/corechecks/containers/docker.go

```diff
@@ -369,12 +360,34 @@ func (d *DockerCheck) Run() error {
 	return nil
 }
 
-func (d *DockerCheck) reportUptime(startTime int64, currentUnixTime int64, tags []string, sender aggregator.Sender) {
+func (d *DockerCheck) reportUptime(startTime, currentUnixTime int64, tags []string, sender aggregator.Sender) {
 	if startTime != 0 && currentUnixTime-startTime > 0 {
 		sender.Gauge("docker.uptime", float64(currentUnixTime-startTime), "", tags)
 	}
 }
 
+func (d *DockerCheck) reportCPUMetrics(cpu *cmetrics.ContainerCPUStats, limits *cmetrics.ContainerLimits, startTime int64, tags []string, sender aggregator.Sender) {
+	if cpu == nil {
+		return
+	}
+
+	sender.Rate("docker.cpu.system", float64(cpu.System), "", tags)
+	sender.Rate("docker.cpu.user", float64(cpu.User), "", tags)
+	sender.Rate("docker.cpu.usage", cpu.UsageTotal, "", tags)
+	sender.Gauge("docker.cpu.shares", float64(cpu.Shares), "", tags)
+	sender.Rate("docker.cpu.throttled", float64(cpu.NrThrottled), "", tags)
```

Comment:
Why not reporting also the `[period](https://github.com/DataDog/datadog-agent/pull/5716/files#diff-725b5ae93959fca9ffbc39b38ca40c47R109)` here

Reply:
I should remove `NrPeriod`, I used it in the first version of the PR (most accurate way to report limits), but I abandoned it as it's only available when a limit is set (and required more work on agent side).

I can add the report of `ThrottledTime` in `docker` though

Reply:
Fixed